Userland/Shell/Builtin.cpp

```diff
@@ -67,6 +67,43 @@ int Shell::builtin_alias(int argc, const char** argv)
     return fail ? 1 : 0;
 }
 
+int Shell::builtin_unalias(int argc, const char** argv)
+{
+    bool remove_all { false };
+    Vector<const char*> arguments;
+
+    Core::ArgsParser parser;
+    parser.set_general_help("Remove alias from the list of aliases");
+    parser.add_option(remove_all, "Remove all aliases", nullptr, 'a');
+    parser.add_positional_argument(arguments, "List of aliases to remove", "alias", Core::ArgsParser::Required::No);
+
+    if (!parser.parse(argc, const_cast<char**>(argv), Core::ArgsParser::FailureBehavior::PrintUsage))
+        return 1;
+
+    if (remove_all) {
+        m_aliases.clear();
+        return 0;
+    }
+
+    if (arguments.is_empty()) {
```

Comment:
Probably should update the command cache as well.

Reply:
Updated, thank you.